R/pkg/NAMESPACE

```diff
@@ -247,6 +247,8 @@ exportMethods("%in%",
               "getField",
               "getItem",
               "greatest",
+              "is_grouping",
```

Comment:
please sort this. you might have forgotten to update this after changing the name